packages/keychain/src/profiles.ts

```diff
@@ -59,30 +59,30 @@ export const registrars = {
   },
 };
 
-export function signProfileForUpload(profile: Profile, keypair: IdentityKeyPair) {
+export async function signProfileForUpload(
+  profile: Profile,
+  keypair: IdentityKeyPair
+): Promise<string> {
   const privateKey = keypair.key;
   const publicKey = keypair.keyID;
 
-  const token = signProfileToken(profile, privateKey, { publicKey });
+  const token = await signProfileToken(profile, privateKey, { publicKey });
```

Comment:
@friedger the function signature for this in blockstack.js is not async?

Reply:
In the profileToken.d.ts of blockstack in my local node_modules folder, the signProfileToken returns a Promise<string> and my IDE was complaining. Any ideas where this comes from?

Reply:
@hstove I will revert the change

Reply:
Looks like it was in an older version of blockstack.js. Resolved.